README

```diff
@@ -95,6 +105,30 @@ curl --user admin:admin -d ip_address=2006:ac81::1104 \
 NOTE: please make sure both the IPv4 and IPv6 addresses are in the trusted
 ip list in iscsi-gateway.cfg.
 
+Erasure Pool Support:
+For the erasure pool, you need to specify the "ecpool=<ec_name>" parameter to store the
+data when creating a disk, and the "pool=<name>" will contiue to be a replicated pool,
+which will store the metadata only.
+
+When creating a disk and disk snapshot, for rest api there has a litte different for
+erasure pool.
+
+You need to use "ecdisk" instead of "disk" in URL:"http://.../disk/...":
+
+curl --user admin:admin -d mode=create -d size=1g -d pool=rbd -d ecpool=ec -d count=5
+     -X PUT http://192.168.122.69:5000/api/ecdisk/rbd/ec/new0_
+curl --user admin:admin -d mode=create -d size=1g -d pool=rbd -d ecpool=ec -d create_image=false
+     -X PUT http://192.168.122.69:5000/api/ecdisk/rbd/ec/new1
+curl --user admin:admin -X GET http://192.168.122.69:5000/api/ecdisk/rbd/ec/new2
+curl --user admin:admin -X DELETE http://192.168.122.69:5000/api/ecdisk/rbd/ec/new3
+
+You need to use "ecdisksnap" instead of "disksnap" in URL:"http://.../disk/...":
+
+curl --user admin:admin -d mode=create
+     -X PUT http://192.168.122.69:5000/api/ecdisksnap/rbd/ec/image/new1
+curl --user admin:admin
+     -X DELETE http://192.168.122.69:5000/api/ecdisksnap/rbd/ec/image/new1
```

Comment:
I may be missing something about REST or flask specifically, but I don't understand the need for a parallel set of EC APIs (`disk` vs `ecdisk` and `disksnap` vs `ecdisksnap`).  Why not just add an optional `ecpool` parameter?  This would mimic the "rbd create" CLI (`--datapool` option).

Reply:
Thinking more about this, I think we should avoid tying this to EC at all.  The actual feature that is being surfaced is a separate data pool.  Let's call the new parameter `datapool` and
- if not present, create a "normal" image
- if present, get the type of the pool
  - if replicated, create with `--datapool=<datapool>`
  - if EC, validate EC pool (bluestore && allow_ec_overwrites=true) and create with `--datapool=<datapool>`

Reply:
Sounds good. Let me try this.